programs/steward/src/lib.rs

```diff
@@ -189,19 +193,20 @@ pub mod steward {
         instructions::close_steward_accounts::handler(ctx)
     }
 
-    /* Passthrough instructions to spl-stake-pool, where the signer is Staker. Must be invoked by `config.authority` */
```

Comment:
I actually like having the visual separation between the above and the passthrough instructions

Reply:
readded